os/arch/arm/src/amebasmart/amebasmart_mipi.c

```diff
@@ -322,12 +325,42 @@ static int amebasmart_mipi_transfer(FAR struct mipi_dsi_host *dsi_host, FAR cons
 	return OK;
 }
 
+#ifdef CONFIG_PM
+static uint32_t rtk_mipi_suspend(uint32_t expected_idle_time, void *param)
+{
+	(void)expected_idle_time;
+	(void)param;
+
+	/* Check anything is needed? */
+	return 1;
+}
+
+static uint32_t rtk_mipi_resume(uint32_t expected_idle_time, void *param)
+{
+	(void)expected_idle_time;
+	(void)param;
+
+	FAR struct amebasmart_mipi_dsi_host_s *priv = &g_dsi_host;
+	/* For MIPI */
+	amebasmart_mipi_init_helper(priv);
+	/* For LCDC */
+	rtl8730e_lcdc_pm();
```

Comment:
It does not cause any screen change like blink or something else at wakeup time?

Reply:
As I have checked, this operation does not affect the last screen shown on LCD (ie. ST7785M has GRAM to buffer LCD data). And I verified LCD operation is working fine after wakeup from PG sleep, with the LCD example.